pyaedt/icepak.py

```diff
@@ -1892,7 +1901,7 @@ def create_fan(
         name : str, optional
             Fan name. The default is ``None``, in which case the default name is used.
         is_2d : bool, optional
-            Whether the fan is modeled as 2D. The default is ``False``, in which
+            Whether the fan is modeled as 2D or not. The default value is ``False``, in which
```

Comment:
```suggestion
            Whether the fan is modeled as 2D. The default is ``False``, in which
```
The former wording is what we are using as standered. Whether implies true or false so the "or not" is not needed. We are not using "default value" but "default".

Reply:
Thanks for the reminder @PipKat .